hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/counter/zookeeper/ZookeeperCounterReporter.java

```diff
@@ -93,7 +93,7 @@ private static TopicName escapedTopicName(String qualifiedTopicName) {
         TopicName topicName = fromQualifiedName(qualifiedTopicName);
         return new TopicName(
                 escapeMetricsReplacementChar(topicName.getGroupName()),
-                escapeMetricsReplacementChar(topicName.getName())
+                topicName.getName()
```

Comment:
Yes, this change is backward compatible. It only touches broken metrics which actually were not read by Hermes (symptoms described here #740). Topics without underscore in names have valid metrics names and nothing will change for them.